### ISSUE TITLE
Style command

### ISSUE THREAD

**Issue by Bharat0112** [okaybenji/text-engine#31]
I have added the following code for stylesheet changing, that changes style by typing stylex. It circles among 3 already given stylesheets. Is there any way to add stylex as a command that takes filename as an argument, so I can add multiple styles. Fro example 'stylex classic' will change the style to classic.css.

`let stylex = () => {
	const selectStylesheet = filename => document.getElementById('styles').setAttribute('href', filename);
const currentStylesheet = document.getElementById('styles').getAttribute('href');
               
			 let newName;

if (currentStylesheet.includes('modern')) {
  newName = 'retro';
} else if (currentStylesheet.includes('retro')) {
  newName = 'classic';
} else {
  newName = 'modern';
}
             println(`You changed the stylesheet to ${newName.toUpperCase()}.css.`);
                selectStylesheet(`styles/${newName}.css`);
};`

**Comment by okaybenji** [comment]
Yeah, that's no problem. Change your function to accept `newName` as an argument like this:

```
let stylex = (newName) => {
  const selectStylesheet = filename => document.getElementById('styles').setAttribute('href', filename);
  println(`You changed the stylesheet to ${newName.toUpperCase()}.css.`);
  selectStylesheet(`styles/${newName}.css`);
};
```

Now you can call the function, passing the filename as a string:

```
stylex('modern');
```

Finally, you'll need to update your game's commands. Since this function accepts a single argument, you'll need to override `commands[1]` like this:

```
commands[1] = Object.assign(commands[0], {stylex});
```

Check out the [Commands](https://github.com/okaybenji/text-engine#commands) section of the documentation for more info.

**Comment by okaybenji** [comment]
Yeah, that's no problem. Change your function to accept `newName` as an argument like this:

```
let stylex = (newName) => {
  const selectStylesheet = filename => document.getElementById('styles').setAttribute('href', filename);
  println(`You changed the stylesheet to ${newName.toUpperCase()}.css.`);
  selectStylesheet(`styles/${newName}.css`);
};
```

Now you can call the function, passing the filename as a string:

```
stylex('modern');
```

Finally, you'll need to update your game's commands. Since this function accepts a single argument, you'll need to override `commands[1]` like this:

```
commands[1] = Object.assign(commands[0], {stylex});
```

Check out the [Commands](https://github.com/okaybenji/text-engine#commands) section of the documentation for more info.
